packages/gatsby-plugin-less/src/gatsby-node.js

```diff
@@ -22,7 +22,7 @@ exports.onCreateWebpackConfig = (
       ? [loaders.null()]
       : [
           loaders.miniCssExtract(),
-          loaders.css({ importLoaders: 2 }),
+          loaders.css({ ...cssLoaderOptions, importLoaders: 2 }),
```

Comment:
@pieh I was gonna update the docs, but I noticed that the feature request was made for `css-loader` and as I understand, you can chain `css-loader` with `less-loader`. I haven't looked into this much, but I am curious as to how this works.

I feel like if I understand this it'll help me update the docs appropriately. Thanks for understanding 😅

Reply:
Holy! That's an in-depth explanation. Still reading it, but just wanted to say I really appreciate it! 🤯